system/views/admin/datamanager/_listingTranslations.cfm

```diff
@@ -0,0 +1,28 @@
+<cfparam name="args.translations"     type="array"  />
+<cfparam name="args.translateUrlBase" type="string" />
+
+<cfoutput>
+	<cfif !arrayIsEmpty( args.translations )>
+		<cfloop index="arrayIndex" from="1" to="3">
```

Comment:
I would expect this to break if there were only two languages (array index out of bounds). Can you verify this?